src/Castle.Core/DynamicProxy/ProxyGenerationOptions.cs

```diff
@@ -92,8 +92,8 @@ public void Initialize()
 				}
 				catch (ArgumentException ex)
 				{
-					throw new InvalidMixinConfigurationException(
-						"There is a problem with the mixins added to this ProxyGenerationOptions: " + ex.Message, ex);
+					throw new InvalidOperationException(
+						"There is a problem with the mixins added to this ProxyGenerationOptions. See the inner exception for details.", ex);
```

Comment:
(In case you're wondering why I'm changing the message: `ArgumentException` messages include the parameter name. When converting this `InvalidOperationException` back to an `ArgumentException` in `DefaultProxyBuilder`, we don't want the original parameter name of `MixinData`'s ctor in the message. If we kept the `+ ex.Message` bit, we'd end up with two parameter names in the final `ArgumentException`'s message.)